index.html

```diff
@@ -1,1090 +1,992 @@
+            
+                      </div>
+                    </div>
+            
+                    <div class="col-auto">
+                      <div class="team-member">
+                        <img class="mx-auto rounded-circle"
+                          src="https://pbs.twimg.com/profile_images/1049501900339437568/fEWp1zUE_400x400.jpg" alt="profile photo">
+                        <h6>Steve Myers</h6>
+            
+                        <ul class="list-inline social-buttons">
+                          <li class="list-inline-item">
+                            <a class="btn btn-dark btn-social mx-2" target="_blank" href="https://github.com/notmandatory">
+                              <i class="fab fa-github"></i>
+                            </a>
+                          </li>
+                          <li class="list-inline-item">
+                            <a class="btn btn-dark btn-social mx-2" target="_blank" href="https://twitter.com/notmandatory">
+                              <i class="fab fa-twitter"></i>
+                            </a>
+                          </li>
+                        </ul>
+                        <div class="bio-mentors"><a target="_blank" href="https://bitcoindevkit.org/">BDK</a></div>
+                      </div>
+                    </div>
+            
+                    <div class="col-auto">
+                      <div class="team-member">
+                        <img class="mx-auto rounded-circle"
+                          src="https://lightning.engineering/static/8d6a850ac0e0c4410f96b1366546415c/a5bc7/jamal.jpg"
+                          alt="profile photo">
+                        <h6>Jamal James</h6>
+                        <ul class="list-inline social-buttons">
+                          <li class="list-inline-item">
+                            <a class="btn btn-dark btn-social mx-2" target="_blank" href="https://github.com/jamaljsr">
+                              <i class="fab fa-github"></i>
+                            </a>
+                          </li>
+                          <li class="list-inline-item">
+                            <a class="btn btn-dark btn-social mx-2" target="_blank" href="https://twitter.com/jamaljsr">
+                              <i class="fab fa-twitter"></i>
+                            </a>
+                          </li>
+                        </ul>
+                        <div class="bio-mentors"><a target="_blank" href="https://github.com/lightningnetwork/lnd">LND</a></div>
+            
+                      </div>
+                    </div>
+            
+            
+                    <div class="col-auto">
+                      <div class="team-member">
+                        <img class="mx-auto rounded-circle"
+                          src="https://pbs.twimg.com/profile_images/1262291843129249792/FUaVdJqx_400x400.jpg" alt="profile photo">
+                        <h6>Carla K Cohen</h6>
```

Comment:
This should be her full name.
```suggestion
                        <h6>Carla Kirk-Cohen</h6>
```